partner_contact_birthdate/__init__.py

```diff
@@ -1,5 +1,5 @@
-# -*- coding: utf-8 -*-
 # Copyright (C) 2014-2015  Grupo ESOC <www.grupoesoc.es>
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © 2017-Apertoso N.V. (<http://www.apertoso.be>)
```

Comment:
This change does not warrant the addition of your copyright IMO. Same goes for all the files.